### DISCUSSION TITLE
ValueError: Connection error, and we cannot find the requested files in the cached path

### DISCUSSION THREAD

**Discussion by GuoqiangJia** [explosion/spaCy#9959] [Help: Installation] [answered]
## I managed to use a third part transformer model, but spacy failed to download the model from huggingface

```
[components.transformer.model]
@architectures = "spacy-transformers.TransformerModel.v3"
name = 'voidful/albert_chinese_tiny'
mixed_precision = false 
```

## Your Environment

* Operating System: Linux 5.4.144+
* Python Version Used: 3.7.12
* spaCy Version Used: 3.2.1
* Environment Information: tensorflow2.7.0, torch1.10.0+cu111

**Comment by polm** [comment]
Can you paste the full error, with stack trace?

If the download failed but the model is actually on the hub, it's possible it was a transient error and you can just try again. I'm not aware of any hub outage at the moment, but occasionally random failures do happen.

**Reply by polm** [reply]
Based on this issue https://github.com/huggingface/transformers/issues/8690, restarting your machine may help.

**Comment by GuoqiangJia** [comment]
Thank you response, please see error detail below, it is something about internet error, but I did training on colab, so I think internet should be OK.
![c3a8736a106a66d8f70ce9eedc01618](https://user-images.githubusercontent.com/77270424/147728381-1eb02053-33da-401f-9828-a80b7d18c21e.png)

**Reply by polm** [reply]
OK, it looks like this is an issue with HuggingFace Transformers, which we don't maintain directly. Take a look at the thread I linked to look for solutions to it - there were some people using colab in that thread too. 

Additionally, please don't paste screenshots of code, paste text as text.

**Reply by GuoqiangJia** [reply]
OK, thanks.

**Comment by GuoqiangJia** [comment]
I replced 'voidful/albert_chinese_tiny' with 'bert-base-uncased', the error was gone. so I think it's possible that spacy can not find model name looking like a path(contains '/') ?

I have resolved the problem by pointing name to my local model directory.

**Comment by GuoqiangJia** [comment, marked as answer]
```
[components.transformer.model]
@architectures = "spacy-transformers.TransformerModel.v3"
name = "D:\\nlp\\textcat_emotion\\albert_chinese_tiny"
mixed_precision = false
```
configurations like above works well.

**Reply by polm** [reply]
Glad you got it working. I'm not sure but I suspect it was either some kind of network issue, unlikely as that may seem, or an issue with the Huggingface cache.
